src/PlasmaShop/CMakeLists.txt

```diff
@@ -125,6 +125,9 @@ add_executable(PlasmaShop WIN32 MACOSX_BUNDLE
                ${pycdc_Headers} ${pycdc_Sources} ${pycdc_GeneratedSources}
                ${PlasmaShop_RCC})
 target_link_libraries(PlasmaShop PSCommon Qt5::Widgets KF5::SyntaxHighlighting)
+if(WIN32)
+    target_link_libraries(PlasmaShop Qt5::WinMain)
```

Comment:
This is supposed to be automatic when `WIN32` is specified in the `add_executable`...  I'd rather find out why it's not working (especially since it works fine for me with the same tool and library versions)...

Reply:
Hmm, reading more on how this is supposed to work, it's supposed to add qtmain (Qt5::WinMain) when you link Qt5::Core, which we really should be linking to explicitly.  I wonder if the difference in generated targets is taking the Qt5::Core link differently?  Either way, I'd bet that adding Qt5::Core to the link list will also resolve this (without the need for an extra WIN32 block).

Reply:
The latest (temporary) fixup commit indicates Qt5::Core alone is insufficient. I think CMP0020 is supposed to address this as well, but, like I say, that seems to have no effect?

Reply:
CMP0020 is already set to NEW because of `cmake_minimum_required(VERSION 3.1)` (this line sets policies older than the required version to NEW automatically)...

Reply:
```
CMake Warning (dev) in src/PlasmaShop/CMakeLists.txt:
  Policy CMP0020 is not set: Automatically link Qt executables to qtmain
  target on Windows.  Run "cmake --help-policy CMP0020" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
OK, I've figured this out. Mostly thanks to the following note from CMake's documentation:
"Call the cmake_minimum_required() command at the beginning of the top-level CMakeLists.txt file even before calling the project() command. It is important to establish version and policy settings before invoking other commands whose behavior they may affect."

I assume that because our `cmake_minimum_required(VERSION 3.1)` came after `project(PlasmaShop)`, `CMP0020` was not being reliably set. Weird. It works now though.